### DISCUSSION TITLE
Are the Camelyon17 models pretrained on ImageNet?

### DISCUSSION THREAD

**Discussion by SebGGruber** [p-lambda/wilds#58] [Q&A] [answered]
Hi!
In section E.2.2 under 'Model.' it is stated that the models are trained from scratch in the case of Camelyon17, but the default in https://github.com/p-lambda/wilds/blob/main/examples/configs/model.py says otherwise. I looked into https://worksheets.codalab.org/worksheets/0x00d14c55993548a1823a710642f6d608 to see if you adjusted the default with a command line argument, but could not find one.
Can you clarify if the statement in the paper is correct?

Thanks a lot!

**Comment by SebGGruber** [comment, marked as answer]
Nevermind. Upon searching further through the directory, I found the spot setting 'pretrained' to FALSE at https://github.com/p-lambda/wilds/blob/5ff3802d20035a07d0629438feda5d1c7d10a088/examples/configs/datasets.py#L44

**Reply by kohpangwei** [reply]
Yup, that's right! Thanks for checking, though.
